apps/settings/src/store/users.js

```diff
@@ -224,11 +224,12 @@ const actions = {
 				cancelToken: searchRequestCancelSource.token,
 			})
 				.then((response) => {
-					if (Object.keys(response.data.ocs.data.users).length > 0) {
+					const usersCount = Object.keys(response.data.ocs.data.users).length
+					if (usersCount > 0) {
 						context.commit('appendUsers', response.data.ocs.data.users)
-						return Object.keys(response.data.ocs.data.users).length >= limit
+						return usersCount
 					}
-					return false
+					return 0
```

Comment:
```suggestion
					return usersCount
```
Combined with the above makes it clearer :)